README.md

```diff
@@ -2,6 +2,8 @@
 
 [[![Join the chat at https://opentx.rocket.chat](https://camo.githubusercontent.com/3d659054abd6ce21c0e47cf3b83a51bda69ca282/68747470733a2f2f64656d6f2e726f636b65742e636861742f696d616765732f6a6f696e2d636861742e737667)](https://opentx.rocket.chat)]()
 
+Target Audience: Developers
+
```

Comment:
Oh.  My point still stands then, which developers are the targets?

**The target audience of this reference guide is Lua script developers.**

Perhaps also with a link to end-user docu if/when exists.